content/en/docs/ambient/usage/l4-policy/index.md

```diff
@@ -75,6 +75,31 @@ Even though the identity of the pod is otherwise correct, the presence of a L7 p
 command terminated with exit code 56
 {{< /text >}}
 
+### Authorization policy attachment
+
+In the simplest enforcement scenario, you want to enforce policy against TCP attributes and you have no waypoint proxies in your traffic's path. These policies can be enforced by ztunnel proxies.
+
+Once you introduce a waypoint proxy, the ideal place to enforce policy shifts. Traffic arriving at the destination ztunnel will be coming from the waypoint's identity because waypoint proxies do not impersonate `src` identity on behalf of the client. This means that even if you only wish to enforce policy against TCP attributes, you should bind that policy to your waypoint proxy. An additional TCP policy may be applied to your workload to request that ztunnel enforce rules like, "in-mesh traffic needs to come from my waypoint in order to reach my application". This type of policy allows you to choose if "bypassing" the waypoint proxy is permissible in your scenario.
+
+The table shown below is based on the following invariants:
+
+1. The source pod is a normal pod which has ztunnel enabled.
+1. Redirection to the waypoint is configured correctly.
+1. The waypoint is configured with the `istio.io/waypoint-for` label set to `service`.
+
+| Waypoint* | Attachment Style | Scope | Source Identity | Enforced By |
+| --- | --- | --- | --- | --- |
+| no | Selector | Pod | client pod | destination ztunnel |
+| yes | Selector | Pod | waypoint | destination ztunnel |
+| no | _empty**_ | Namespace | client pod | destination ztunnel |
+| yes | _empty**_ | Namespace | waypoint | destination ztunnel |
+| yes | `targetRefs` | Service | client pod | waypoint |
+| yes | `targetRefs` | Gateway | client pod | waypoint |
+
+* Whether or not there is already a waypoint is in the traffic path.
+
+** If no Selector or `targetRef` is specified the policy is Namespace scoped.
```

Comment:
```suggestion
| Waypoint † | Attachment Style | Scope | Source Identity | Enforced By |
| --- | --- | --- | --- | --- |
| no | Selector | Pod | client pod | destination ztunnel |
| yes | Selector | Pod | waypoint | destination ztunnel |
| no | _empty ‡_ | Namespace | client pod | destination ztunnel |
| yes | _empty ‡_ | Namespace | waypoint | destination ztunnel |
| yes | `targetRefs` | Service | client pod | waypoint |
| yes | `targetRefs` | Gateway | client pod | waypoint |

† Whether or not there is a waypoint in the traffic path.
‡ If no Selector or `targetRef` is specified, the policy is namespace scoped.
```

There are many ways this table could have its columns ordered/sorted.  Was there a reason this one was chosen? (Do we offer sortable tables in our docs?)